src/pages/Create.tsx

```diff
@@ -1,42 +1,51 @@
-
+import { useState } from "react";
 import { useSelector } from "react-redux";
 import { apiReducerState } from "../redux/reducers/apikey";
 import API from "../util/Api";
+import './Create.scss'
+
+const UIC_DOMAIN = 'uic.edu';
+
+const InputBar = ({placeholder, inputText, setText} 
+    : {placeholder: string, inputText: string, setText: React.Dispatch<React.SetStateAction<string>>}) => {
+    return (
+        <input type="text" 
+            value={inputText}
+            onChange={event => setText(event.target.value)}
+            placeholder={placeholder}
+            className="input-bar"/>
+        );
+}
```

Comment:
This is very much overkill here, we only need onChange to reference a javascript variable and doesn't need to dynamically update the page. Check below.